site/en/docs/webstore/cws-dashboard-distribution/index.md

```diff
@@ -69,7 +75,7 @@ distribution page of the developer console:
 {% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/zpV6tBHuf3HLBpRjGsNA.png", alt="Screenshot showing
 the domain publishing option in the developer console", width="643", height="200", class="screenshot" %}
 
-See [Enterprise publishing options][domain-publishing-overview] for an overview of this and other
+See [Enterprise publishing options][2] for an overview of this and other
```

Comment:
Going forward, note that we prefer the text links over the numeric ones. And listed at the end alphabetically. Don't worry for this PR.

Reply:
You know I will worry... I will change them all!!! 💪